runtime/doc/syntax.txt

```diff
@@ -4473,19 +4473,19 @@ Notes:
   matched.  This doesn't work: "a\nb"ms=e.  You can make the highlighting
   start in another line, this does work: "a\nb"hs=e.
 
-Example (match a comment but don't highlight the /* and */): >
+Example (match a comment but don't highlight the `/* and */`): >
   :syntax region Comment start="/\*"hs=e+1 end="\*/"he=s-1
-<
```

Comment:
actually, I'd suggest keeping this as two separate code blocks, so we can (in the future) mark the first as `vim`